app/controllers/answers_controller.rb

```diff
@@ -14,6 +14,8 @@ def create
     @answer.user = current_user
 
     if @answer.save
+      QuestionMailer.asker_mail(@question).deliver_now
+      QuestionMailer.members_mail(@question).deliver_now
```

Comment:
Please use `deliver_later` to send out the emails after the request is done. That way the site responds faster.